package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "node-js-playwright-browserstack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "sample-test": "npx browserstack-node-sdk playwright test --config=./playwright.config.js",
+    "postinstall": "npm update browserstack-node-sdk"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@playwright/test": "latest",
+    "browserstack-node-sdk": "latest"
```

Comment:
do not keep this as latest.
lets fix this to whatever is the version for which we have verified it is working.

also update package-lock.json accordingly.